docs/usage/self-hosted-configuration.md

```diff
@@ -15,6 +15,22 @@ Please also see [Self-Hosted Experimental Options](./self-hosted-experimental.md
 
 ## allowCustomCrateRegistries
 
+## allowExtraConfig
+
+This option is only available via command line option `--allow-extra-config`.
+
+You can allow custom configurations in the `renovate.json` by using this command line option.
+
+Example:
+
+```bash
+yarn start --allow-extra-config
+```
+
+This option is useful for those who wants to add custom configurations.
+
+The extraneous options won't be used by the renovate as they are not understood by Renovate.
```

Comment:
I really don't know what this means:

> The extraneous options won't be used by the renovate as they are not understood by Renovate.

It feels like we're missing something that explains what _extraneous options_ means.